packages/page-bounties/src/BountyCuratorProposedActions.tsx

```diff
@@ -0,0 +1,47 @@
+// Copyright 2017-2020 @polkadot/app-bounties authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { AccountId } from '@polkadot/types/interfaces';
+
+import React, { useMemo } from 'react';
+
+import { TxButton } from '@polkadot/react-components';
+import { useAccounts, useApi } from '@polkadot/react-hooks';
+
+import { useTranslation } from './translate';
+
+interface Props {
+  curatorId: AccountId;
+  index: number;
+}
+
+function BountyCuratorProposedActions ({ curatorId, index }: Props) {
+  const { t } = useTranslation();
+  const { api } = useApi();
+  const { allAccounts, hasAccounts } = useAccounts();
```

Comment:
```suggestion
  const { allAccounts } = useAccounts();
```

... along with `isCurator` change

Reply:
Sorry, I did the force push :(